api/src/hyp3_api/handlers.py

```diff
@@ -28,7 +27,7 @@ class QuotaError(Exception):
 def post_jobs(body, user):
     print(body)
     if not context['is_authorized']:
-        abort(403)
+        return problem(403, 'Forbidden', 'You do not have permission to submit jobs.')
```

Comment:
maybe `User  {user} does not have permission to submit jobs.` would be better?  I imagine mixing up EDL usernames will be somewhat common, at least internally at ASF.

Reply:
I agree, putting the username in there would be best.